CHANGELOG.md

```diff
@@ -9,6 +9,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ### Changed
 
+- Update to `tokio` 1.0
```

Comment:
Can you expand here on in which cases this is a breaking change? I think only when setting StatsConfig.handle, and that otherwise this is transparent.